uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/quotes/BookingQuotesPresenter.kt

```diff
@@ -20,4 +27,45 @@ class BookingQuotesPresenter(val view: BookingQuotesMVP.View) : BookingQuotesMVP
     override fun capitalizeCategory(category: String) {
         view.setCategoryText(category.capitalize(Locale.getDefault()))
     }
+
+    override fun createTagsString(tags: List<String>, shortVersion: Boolean): Spannable {
+        val tagsText = SpannableStringBuilder("")
+
+        tags.forEachIndexed { index, tag ->
+            val span: Spannable = SpannableString("  ${tag.capitalize(Locale.ROOT)} ")
+            val icon = getTagIcon(tag)
+            val image = icon?.let {
+                it.setBounds(0, 0, (it.intrinsicWidth / INTRINSIC_BOUND_FACTOR).toInt(), (it.intrinsicWidth / INTRINSIC_BOUND_FACTOR).toInt())
+                ImageSpan(it, ImageSpan.ALIGN_BASELINE)
+            }
+            image?.let {
+                span.setSpan(image, 0, 1, Spannable.SPAN_INCLUSIVE_EXCLUSIVE)
+            }
+
+            tagsText.append(span)
+
+            if (shortVersion && index == 1) {
+                tagsText.append(" + ").append((tags.size - 1 - index).toString())
+                return tagsText
+            }
+        }
+
+        return tagsText
+    }
+
+    private fun getTagIcon(tag: String): Drawable? {
+        return when (tag.toLowerCase(Locale.ROOT)) {
+            "executive" -> view.getDrawableResource(R.drawable.kh_uisdk_ic_executive)
```

Comment:
Would it be better if we explicitly prefix them with `tag` them `kh_uisdk_ic_tag_`

Reply:
Done